fo4_database.csv

```diff
@@ -33753,6 +33753,7 @@ id,fo4,vr,status,name
 570988,0x1436d4a68,0x1436faa90,2,DAT_1436d4a68
 571019,0x142c4fba8,0x142c86588,3,MagicItem_2
 571036,0x1400a7630,0x1400a8240,2,BGSLoadGameBuffer::LoadFormID<TESTopicInfo>(TESTopicInfo*&)
+571037,0x14270d550,0x1426dbc40,2,BSScript::NF_util::NativeFunctionBase::Call(BSTSmartPointer&lt;BSScript::Stack,BSTSmartPointerIntrusiveRefCount&gt;_&amp;,BSScript::ErrorLogger_&amp;,BSScript::Internal::VirtualMachine_&amp;,bool)
```

Comment:
BTW, just realized after release that this is a malformed csv entry. It should have been quoted. Otherwise you've just added 4 columns.

Reply:
ahh sorry I did not even notice there were commas in there.

I'll quote that string in the column to avoid that in the future.